templates/docs/examples/patterns/card/header.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
 <div class="p-card">
-  <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/dca2e4c4-raspberry-logo.png" alt="" />
+  <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/8fdc8250-Ubuntu+and+Raspberry+Pi.svg" alt="" />
```

Comment:
Why the image change?

Reply:
@bartaz  Updated image looks better on dark theme as it has no dark text